config/crd/bases/postgresql.easymile.com_postgresqluserroles.yaml

```diff
@@ -0,0 +1,147 @@
+          API.
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: PostgresqlUserRoleSpec defines the desired state of PostgresqlUserRole.
+            properties:
+              importSecretName:
+                description: Import secret name
+                type: string
+              mode:
+                description: User mode
+                enum:
+                - PROVIDED
+                - MANAGED
+                type: string
+              privileges:
+                description: Privileges
+                items:
+                  properties:
+                    database:
+                      description: Postgresql Database
+                      properties:
+                        name:
+                          description: Custom resource name
+                          type: string
+                        namespace:
+                          description: Custom resource namespace
+                          type: string
+                      required:
+                      - name
+                      type: object
+                    generatedSecretName:
+                      description: Generated secret name prefix
+                      minLength: 1
+                      type: string
+                    privilege:
+                      description: User privileges
+                      enum:
+                      - OWNER
+                      - WRITER
+                      - READER
+                      type: string
+                  required:
+                  - database
+                  - generatedSecretName
+                  - privilege
+                  type: object
+                type: array
+              rolePrefix:
+                description: User role prefix
+                type: string
+              userPasswordRotationDuration:
+                description: User password rotation duration
+                type: string
+              workGeneratedSecretName:
+                description: Simple user password tuple generated secret name
+                type: string
+            required:
+            - privileges
```

Comment:
Mode is also required

Reply:
It is generated so... But Enums are required :)